source/features/extend-diff-expander.css

```diff
@@ -19,3 +19,29 @@
 	);
 	cursor: pointer;
 }
+
+/* Support new Commit view */
+.rgh-extend-diff-expander .diff-line-row:has(button[data-direction]):hover {
+	color: var(
+		--diffBlob-hunkNum-fgColor-hover,
+		var(--fgColor-onEmphasis, fuchsia)
+	);
+	background: var(
+		--diffBlob-hunkNum-bgColor-hover,
+		var(--bgColor-accent-emphasis, fuchsia)
+	);
+	cursor: pointer;
+	transition: var(--duration-fast) var(--easing-easeInOut);
```

Comment:
What is this for? I don't see any transitions

Reply:
Look at the left, you can see its difference.

https://github.com/user-attachments/assets/a5b9f3be-85b7-4112-ac7a-2a18366dfc19


https://github.com/user-attachments/assets/7f5a66ad-041e-4488-8e46-55441c38a1ce